index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="index.css" />
+  <title>Awesome Books</title>
+</head>
+<body>
+  <nav class="nav-container">
+    <div class="sub-nav-left">
+      <h1 class="title-text">Awesome Books</h1>
+      <p class="list-text">List</p>
+    </div>
+    <div class="sub-nav-right">
+      <p class="add-new-text deactive">Add new</p>
+      <p class="contact-text deactive">Contact</p>
+    </div>
+  </nav>
+
+  <div class="container">
+    <h1 class="title-text">Awesome Books</h1>
+    <p class="page-intro">Discover and explore our collection of awesome books</p>
```

Comment:
- [ ] As mentioned by the previous :point_up: reviewer it is very important not to have more than one `h1` in an entire HTML document, and this is for accessibility reasons. As you might know, a `h1` tag suggests the importance of a heading but this will no longer be so if it's overused. So, I'll suggest that you change this or the one on line `13` since it's a logo but still, this h1 is there so kindly replace it with other `hx` tags:slightly_smiling_face:

Reply:
`h1` fixed